### ISSUE TITLE
'no kernel named ...', problem with conda env's

### ISSUE THREAD

**Issue by arsenovic** [spatialaudio/nbsphinx#90]
when i try to execute notebooks during the sphinx build with   `nbsphinx_execute = 'always'` in `conf.py` i get 
                                                                        
    Notebook error:
    NoSuchKernel in tutorials/Calibration.ipynb:
    No such kernel named conda-env-py35-py
    make: *** [html] Error 1

i am using  conda, and am running sphinx build from a conda env. i have tried to change the kernel setting in the offending notebook to many different options, the obvious first choice was the env i am running sphinx build in. they all give errors. 

it seems to indicate the attempt to set the kernel is failing.

**Comment by mgeier** [comment]
The first thing you should try is if you can execute it with `nbconvert`:

```
python3 -m nbconvert --to html --execute tutorials/Calibration.ipynb
```

(or something similar, I didn't try this command)

Then you should look at the notebook metadata, especially the `kernelspec` section.

**Comment by arsenovic** [comment]
this mentions the `kernel_name` argument, http://nbconvert.readthedocs.io/en/latest/execute_api.html#executing-notebooks-using-the-python-api-interface

**Comment by mgeier** [comment]
This should be resolved by #98.
